docs/dev/search_api.rst

```diff
@@ -4,15 +4,34 @@
 Search API
 ==========
 
-The search supports both ``GET`` and ``POST``.
+SearXNG supports querying via a simple HTTP API.
+Two endpoints, ``/`` and ``/search``, are supported for both GET and POST methods.
+The GET method expects parameters as URL query parameters, while the POST method expects parameters as form data.
 
-Furthermore, two endpoints ``/`` and ``/search`` are available for querying.
+If you want to consume the results as JSON, CSV, or RSS, you need to set the
+``format`` parameter accordingly. Supported formats are defined in settings.yml, under the ``search`` section.
+Requesting an unset format will return a 403 Forbidden error. Be aware that many public instances have these formats disabled. 
+You can still get HTML-formatted results and parse them yourself.
```

Comment:
I wouldn't advertise scraping SearXNG instances in our documentation.

If instance admins don't want to enable the API, they probably also don't want other to scrape results from their instances.

Reply:
Yeah I agree. Let's say that if you are reading SearXNG's docs about the search API  you're probably smart enough to figure this out by yourself, but I'll cut the last part nonetheless :smile: